apps/site/pages/docs/component-customization/creating-component.mdx

```diff
@@ -0,0 +1,69 @@
+---
+title: 'Creating a Component'
+sidebar_label: 'Getting Started'
+---
+
+import { Callout } from 'nextra-theme-docs'
+
+<header>
+
+# Creating a Component
+
+</header>
+
+<Callout type="warning" emoji="⚠️">
+This documentation is currently under development.
+</Callout>
+
+In this guide, you'll learn how to create a component from scratch and use it in your FastStore project. For the purpose of this guide, we'll create a `Price` component to display the price of a product in the `Product Details` component.
+
+|   |   |
+|---|---|
+| ![faststore-component](https://vtexhelp.vtexassets.com/assets/docs/src/faststore-price-component___d9d0e2fbf65e6680d0f26bed7a0e9b8e.png) The **FastStore `Price` component** displays the price of a given product, discount and total values.| ![custom-component](https://vtexhelp.vtexassets.com/assets/docs/src/custom-price-component___3b8a888867b6bf354541236163ef3d79.png) The **custom `Price` component** created can display any information that you set. In this case it renders the name `Price`|
```

Comment:
Let's change this images pls 🙈 
the `QuantitySelector` and buy button don't look good here.

Reply:
Yes, pls! hehe
Can you help me with that, Fanny? Since the issue with the symlink on Windows, I'm unable to run the starter on my machine 😕

Reply:
sure!! DM me :)